src/ios/Common/OSFANLConsentHelper.swift

```diff
@@ -0,0 +1,66 @@
+import FirebaseAnalytics
+import FirebaseCore
+
+@objc enum ConsentTypeRawValue: Int {
```

Comment:
Instead of adding a new `stringValue` variable, we should conform the enum to `CustomStringConvertible` and implement the `description` property.